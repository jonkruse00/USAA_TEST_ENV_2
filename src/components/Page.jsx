import React from 'react';
import { WELCOME_MSG } from '../const/Greetings';

import './Page.less';

if (test) {
  console.log('tww');
} 
else {
  console.log('tww');
}

const Page = () => (
  <div className="button-wrapper">
    {WELCOME_MSG.heading}
    <div className="link-button">
      Button
    </div>
    {WELCOME_MSG.footer.main}
  </div>
);

export default Page;
