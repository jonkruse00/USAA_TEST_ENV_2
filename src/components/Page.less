.MuiPaper-root.filter-row {
  -webkit-box-shadow: 1px 9px 50px -25px @primaryCyan;
  -moz-box-shadow: 1px 9px 50px -25px @primaryCyan;
  box-shadow: 1px 9px 50px -25px @primaryCyan;

  .button-wrapper {
    text-align: center;

    .MuiButton-root:hover {
      background-color: unset;
    }

    .link-button {
      margin: 1em;

      .MuiButton-label {
        text-transform: capitalize;
        color: @primaryCyan;

        &:hover {
          text-decoration: underline;
        }
      }
    }
  }
}
