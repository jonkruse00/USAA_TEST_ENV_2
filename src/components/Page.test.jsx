import React from 'react';
import renderer from 'react-test-renderer';
import Page from './Page';

describe('Page', () => {
  it('should render Page component', () => {
        const tree = renderer.create(<Page />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});