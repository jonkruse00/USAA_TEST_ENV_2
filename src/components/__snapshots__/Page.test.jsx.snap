// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page should render Page component 1`] = `
<div
  className="button-wrapper"
>
  Welcome goes here!
  <div
    className="link-button"
  >
    Button
  </div>
  Main Footer
</div>
`;
